src/ui/components/Scanner/Scanner.tsx

```diff
@@ -126,6 +127,21 @@ const Scanner = forwardRef(
         return;
       }
 
+      if (
+        [
+          OperationType.SCAN_SSI_BOOT_URL,
+          OperationType.SCAN_SSI_CONNECT_URL,
+        ].includes(currentOperation)
+      ) {
+        if (OperationType.SCAN_SSI_BOOT_URL === currentOperation)
+          dispatch(setBootUrl(content));
```

Comment:
I'm not a fan of skipping the brackets with if statements here because it can lead to bugs later - https://stackoverflow.com/a/2125078